requirements.pip

```diff
@@ -0,0 +1,8 @@
+ipython
```

Comment:
This looks handwritten, make sure you generate this by running `pip freeze > requirements.pip`